io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,410 @@
+        sha256: f1003466ad074e9b0c8d421a204121100b0751c96fc6fcf3d1456bd12f8a00a1
+
+  - name: bdw-gc
+    buildsystem: autotools
+    config-opts:
+      - --disable-debug
+      - --disable-docs
+      - --disable-dependency-tracking
+    sources:
+      - type: git
+        url: https://github.com/ivmai/bdwgc
+        tag: v8.2.4
+
+  - name: guile
+    buildsystem: autotools
+    config-opts:
+      - --with-libltdl-prefix=/app
+      - --with-libreadline-prefix=/app
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile/guile-2.2.7.tar.xz
+        sha256: cdf776ea5f29430b1258209630555beea6d2be5481f9da4d64986b077ff37504
+
+  - name: g-wrap
+    buildsystem: autotools
+    config-opts:
+      - --disable-Werror
+    sources:
+      - type: archive
+        url: http://savannah.nongnu.org/download/g-wrap/g-wrap-1.9.15.tar.gz
+        sha256: 0ff6e2700e74b457323726f7c2551a466d8ba44339705f6792d7b533145c602a
+
+  - name: guile-cairo
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/guile-cairo/guile-cairo-1.10.0.tar.gz
+        sha256: ae3d7e0f0a8ae93d0a15c692514eb30f2478d407052064976fc59a291cccdd5c
+
+  - name: guile-lib
+    buildsystem: autotools
+    config-opts:
+      - --with-guile-site=yes
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/guile-lib/guile-lib-0.2.7.tar.gz
+        sha256: e4ef3b845f121882c7c0cf04f81a1cb8fd360c6f64b56b868de5546214f904de
+
+  - name: guile-gnome-platform
+    buildsystem: autotools
+    build-options:
+      - cflags: "-Wall"
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile-gnome/guile-gnome-platform/guile-gnome-platform-2.16.5.tar.gz
+        sha256: 298d8c4f9b567bfe87beda18ed58d047c2e01b88c80895129de5466b921ccebe
+
+  - name: gemmi
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/project-gemmi/gemmi.git
+        tag: v0.6.7
+
+  - name: Catch2
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - "-DCMAKE_CXX_STANDARD=17"
+    sources:
+      - type: archive
+        url: https://github.com/catchorg/Catch2/archive/refs/tags/v3.7.1.tar.gz
+        sha256: c991b247a1a0d7bb9c39aa35faf0fe9e19764213f28ffba3109388e62ee0269c
+
+  - name: eigen
+    buildsystem: simple
+    config-opt:
+      -Dpkg_config_libdir=/app/lib/pkgconfig
+      -DCMAKE_INSTALL_PREFIX=/app
+    build-commands:
+      - cmake -S . -B eigen-build
+      - cmake --install eigen-build --prefix=/app
+    post-install:
+      - install -D cmake/FindEigen3.cmake /app/share/cmake/Modules/FindEigen3.cmake
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.3.9/eigen-3.3.9.tar.gz
+        sha256: 7985975b787340124786f092b3a07d594b2e9cd53bbfe5f3d9b1daee7d55f56f
+
+  - name: maeparser
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/maeparser
```

Comment:
I already mentionned that it should install install in `/app/tmp`.

It's has a cmake build system. Use that. Then if it is build only, you can add a cleanup rule with `'*'` in the module.

Same for the others.

Reply:
> It's has a cmake build system. 
I know that.

There modules should be built in following RDKit module.

Nonetheless, the entire build fails.

Reply:
Redolved. 

https://github.com/flathub/flathub/pull/5690?notification_referrer_id=NT_kwDOAUFjg7QxMjc2ODI2MzY2MzoyMTA2MjUzMQ#discussion_r1794837739